### DISCUSSION TITLE
2.13.0

### DISCUSSION THREAD

**Discussion by BrianLima** [BrianLima/UWPHook#102] [General]
Hey people, a few fixes for today:

- Fix the persistent icons for games with encrypted icons that could not be copied
- Fix some games missing from the app list
- Properly handle apps that weren't being handled well in version 2.12
- Add the ability to generate a log file
- Change the default installation from %PROGRAMFILES% to %APPDATA%, to avoid admin issues on launching games

Thanks @stevealexandre for the recent contributions 🎉 

If you face bugs or have questions, head over to our subreddit at http://reddit.com/r/UWPHook or ping me at our issues or my Twitter.

This software is open-source under the MIT License.

If you like what i did with it and want to support me, you can cheer me up at my Twitter or [pay me a coffee via Paypal](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=9YPV3FHEFRAUQ), it will help me to continue to build amazing open source tools for you!

Thanks for your support, and game on!

<hr /><em>This discussion was created from the release <a href='https://github.com/BrianLima/UWPHook/releases/tag/2.13.0'>2.13.0</a>.</em>

**Comment by jbiribi** [comment]
I'm on the lastest version, but games aren't launching on steam.

**Comment by Naydaze** [comment]
Ii added both the Minecraft windows 10 and Minecraft launcher which do work but when I try to play them using stem link it just goes to a black screen and the app opens into another window which I cannot access using steamlink.

**Comment by shep2363** [comment]
i ran the .exe file on my steamdeck but i can't seem to find the program

**Reply by BrianLima** [reply]
UWPHook does not work with Steam Deck
